src/ontology/mondo-edit.obo

```diff
@@ -584963,6 +584963,21 @@ is_a: MONDO:0003767 {source="NCIT:C50888", source="https://orcid.org/0009-0009-6
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8003" xsd:anyURI
 
+[Term]
+id: MONDO:1030011
+name: paroxysmal atrial fibrillation
+def: "A disorder characterized by intermittent atrial fibrillation that lasts less than seven days from onset." [https://orcid.org/0009-0009-6791-365X]
```

Comment:
Definition:
This definition implies that "any disorder" that has continuous atrial fibrillation that lasts for more than seven days is a "persistent atrial fibrillation". This is not correct.
The definition (genus-differentia) should use the most specific genus, which here should be "atrial fibrillation" (ie the parent term).

dbxref:

Please add the PMID of the paper shared in the issue as a dbxref for this term.
this ORCID refers to a private record. Is this the correct orcid? If so, who's orcid is it?

Reply:
updated